source/backend/api/Models/Concepts/AcquisitionFile/AgreementModel.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+
+namespace Pims.Api.Models.Concepts
+{
+    public class AgreementModel : BaseAppModel
+    {
+        public long AgreementId { get; set; }
+
+        public long AcquisitionFileId { get; set; }
+
+        public TypeModel<string> AgreementType { get; set; }
+
+        public DateTime? AgreementDate { get; set; }
+
+        public bool? AgreementStatus { get; set; }
+
+        public DateTime? CompletionDate { get; set; }
+
+        public DateTime? TerminationDate { get; set; }
+
+        public DateTime? CommencementDate { get; set; }
+
+        public decimal? DepositAmount { get; set; }
+
+        public int? NoLaterThanDays { get; set; }
+
+        public decimal? PurchasePrice { get; set; }
```

Comment:
right, so I guess this should be non-nullable as per confluence.

Reply:
Until I get a response from the BAs I will merge this and fix it if they requrie it.